common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +99,840 @@ innovation_taming_the_skies = {
+			this = culture:laughing_skull
+		}
+	}
+
+	unlock_maa = savage
+
+	flag = global_maa
+}
+
+innovation_sentinel = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @sentinel
+
+	potential = {
+		OR = {
+			this = culture:night_elf
+			this = culture:highborne
+		}
+	}
+
+	unlock_maa = sentinel
+
+	flag = global_maa
+}
+
+innovation_snow_hunter = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @snow_hunter
+
+	potential = {
+		this = culture:taunka
+	}
+
+	unlock_maa = snow_hunter
+
+	flag = global_maa
+}
+
+innovation_swarm = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @swarm
+
+	potential = {
+		this.culture_group = culture_group:aqir_group
+	}
+
+	unlock_maa = swarm
+
+	flag = global_maa
+}
+
+innovation_thornguard = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @thornguard
+
+	potential = {
+		this = culture:quilboar
+	}
+
+	unlock_maa = thornguard
+
+	flag = global_maa
+}
+
+innovation_tidehunter = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @tidehunter
+
+	potential = {
+		this.culture_group = culture_group:murloc_group
+	}
+
+	unlock_maa = tidehunter
+
+	flag = global_maa
+}
+
+
+innovation_ursa = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @ursa
+
+	potential = {
+		this = culture:furbolg
+	}
+
+	unlock_maa = ursa
+
+	flag = global_maa
+}
+
+innovation_warbrave = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @warbrave
+
+	potential = {
+		this.culture_group = culture_group:tauren_group
+	}
+
+	unlock_maa = warbrave
+
+	flag = global_maa
+}
+
+innovation_war_dinosaur = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dinos
+
+	potential = {
+		this = culture:zandalari
+	}
+
+	unlock_maa = war_dinosaur
+
+	flag = global_maa
+}
+
+innovation_wolfrider = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @wolfrider
+
+	potential = {
+		this.culture_group = culture_group:orc_group
+	}
+
+	unlock_maa = wolfrider
+
+	flag = global_maa
+}
+
+
+####Early Medival
+
+innovation_baradian_marine = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @marines
+
+	potential = {
+		OR = {
+			this = culture:tirassian
+			this = culture:baradin
+		}
+	}
+
+	unlock_maa = baradian_marine
+
+	flag = global_maa
+}
+
+
+innovation_blackrock_legionnaire = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @legionnaire
+
+	potential = {
+		this = culture:blackrock
+	}
+
+	unlock_maa = blackrock_legionnaire
+
+	flag = global_maa
+}
+
+innovation_chaosborn = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @chaosborn
+
+	potential = {
+		OR = {
+			this = culture:eredar
+			this = culture:eredruin
+		}
+	}
+
+	unlock_maa = chaosborn
+
+	flag = global_maa
+}
```

Comment:
What these 2 triggers supposed to mean?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/134934620-450c0d4d-b783-41fb-bba8-9c954c887cc6.png)
![image](https://user-images.githubusercontent.com/46384992/134934653-25adf951-7c42-4c29-ae15-06e1bacfcba9.png)

</details>

They have basically the same name.

`has_massive_infantry_maa_trigger` used only in this place.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/134934755-9504524a-9ffc-405d-8402-7961639dff55.png)

</details>

From what I see, it needs refactoring.

Reply:
The top one is meant for if a culture should have giants overall (special maa included), the bottom one is for if they specifically have a special maa.

Reply:
> The top one is meant for if a culture should have giants overall (special maa included), the bottom one is for if they specifically have a special maa.

Why it's not named to reflect it?
Why `has_massive_infantry_maa_trigger` isn't put inside `has_giant_maa_trigger` if it's included?

Reply:
Will rename

Reply:
> Why has_massive_infantry_maa_trigger isn't put inside has_giant_maa_trigger if it's included?
I think this would cause a logical error in this context no?
![image](https://user-images.githubusercontent.com/74790101/135362617-3ee51079-496a-4d85-a027-a50c97a22847.png)

Reply:
> I think this would cause a logical error in this context no?

Why?